services/dy-static-file-server/tests/unit/test_dy_static_file_server_inputs_to_outputs.py

```diff
@@ -163,6 +163,7 @@ def test_folder_mirror(
     output_dir: Path,
     create_files_in_input: Callable,
     key_values_json_outputs_content: str,
+    env_server_root: None,
```

Comment:
MINOR: ``env_server_root: Optional[??] = None``

Reply:
No, that is correct. The fixture returns nothing.